libparsec/crates/types/src/error.rs

```diff
@@ -75,6 +75,12 @@ pub enum DataError {
 
     #[error("Invalid version: expected `{expected}`, got `{got}`")]
     UnexpectedVersion { expected: u32, got: u32 },
+
+    #[error("Invalid file content")]
+    InvalidFileContent,
```

Comment:
`InvalidFileContent` seems very broad... but still restricted to file (so not *that* broad)

What about `InconsistentContent` ?

Or, if we want to have a specific error for file (but I'd rather not since we didn't have errors for specific manifests so far), `InconsistentFileContent` ? (given invalid is used everywhere we don't know much about the cause of the error, while inconsistent show that different data within the manifest are in conflict)